xml/hardware_nvdimm.xml

```diff
@@ -38,33 +39,29 @@
   <para>
    Like SSDs, NVDIMMs provide non-volatile storage: their contents are retained
    when the system is powered off or restarted. Also like SSDs, sector-level
-   access is also possible if that is more suitable for a particular
-   application.
+   access is possible if that is more suitable for a particular application.
   </para>
 
   <para>
    This new storage subsystem is defined in version 6 of the ACPI standard.
-   However, some types of NVDIMMs already existed before the standard was
-   defined. These can be used in the same way.
-  </para>
-  
-  <para>
-   Different models use different forms of electronic storage medium, such as
-   Intel 3D XPoint, or a combination of NAND flash and DRAM, and in future,
-   memristor storage or other forthcoming types of non-volatile memory.
+   However, NVDIMMs already existed before the standard was defined, meaning
+   that non-compliant hardware exists. These can be used in the same way.
   </para>
 
   <para>
-   This means that different vendors' NVDIMMs offer different performance and
-   durability characteristics. Typically, they are slower than DRAM, but
-   considerably faster than SSDs based on NAND-flash technology.
+   Different models use different forms of electronic storage medium, such as
+   Intel 3D XPoint, or a combination of NAND flash and DRAM. New forms of
+   non-volatile RAM are also in development. This means that different vendors'
+   NVDIMMs offer different performance and durability characteristics.
+   Typically, they are slower than DRAM, but considerably faster than SSDs based
```

Comment:
Same problem here: 'they' could either refer to the 'durability characteristics' or to 'NVDIMM'. For experts, this is obvious but maybe not for the translators. Rephrase?